src/DropDown/DropDown.tsx

```diff
@@ -19,31 +20,23 @@ export interface Item {
 export interface Props {
   children: React.ReactNode;
   items: Item[];
+  initialSelectedItem: Item;
 }
 
 const itemToString = (item: Item): string => (item ? item.value : '');
 
 export const DropDown = (props: Props): JSX.Element => (
   <div>
-    <Downshift {...props} itemToString={itemToString}>
-      {({ getMenuProps, getItemProps, getToggleButtonProps, highlightedIndex, selectedItem, isOpen }) => (
-        <div>
+    <Downshift {...props} initialSelectedItem={props.initialSelectedItem} itemToString={itemToString}>
+      {({ getMenuProps, getItemProps, getToggleButtonProps, highlightedIndex, selectedItem, isOpen, getRootProps }) => (
+        <StyledDownshiftWrapper {...getRootProps()}>
           <StyledDownshiftPreview {...getToggleButtonProps()}>
-            {selectedItem ? (
-              <StyledDownshiftPreviewInner>
-                {selectedItem.thumbnail && (
-                  <StyledListItemImage alt={`${selectedItem.label} - thumbnail`} src={selectedItem.thumbnail} />
-                )}
-                {selectedItem.label}
-              </StyledDownshiftPreviewInner>
-            ) : (
-              <StyledDownshiftPreviewInner>
-                {props.items[0].thumbnail && (
-                  <StyledListItemImage alt={`${selectedItem.label} - thumbnail`} src={props.items[0].thumbnail} />
-                )}
-                {props.items[0].label}
-              </StyledDownshiftPreviewInner>
-            )}
+            <StyledDownshiftPreviewInner>
```

Comment:
You've removed the `selectedItem ?` check. Any reason for it?

Reply:
Yup. So, previously I was using the ternary to show the selected item or the first item in the `items` array. Now I have added the `initialSelectedItem` there is never a time when it will get to the `else` part of the ternary.